ProGen/rebuildall32.bat

```diff
@@ -9,11 +9,8 @@ mkdir ProGen\log
 
 call ProGen\progen.bat Win32 >ProGen\log\progen32.log 2>&1
 rd /q /s lib bin     & call buildwin  90 rebuild all both Win32 samples tests devenv  > ProGen\log\90.x32.log 2>&1
```

Comment:
Should the "90"(Visual Studio 2008) project files be removed also? Same applies to the .gitignore.

Reply:
I do not know. See these for more info.

https://github.com/pocoproject/poco/pull/4342#issuecomment-1854714427
https://github.com/pocoproject/poco/pull/4323#pullrequestreview-1766477057

Reply:
I see. @aleks-f would know.

Reply:
The only purpose for the VS90 projects is use  as a template by ProGen - it launches the VS conversion tool and gives it VS90 project file as a template to generate a later VS project versions (with some "massaging" in between). Nothing is built with VS90 projects anymore.